<launch>
    <group ns="vehicle0">
        <node pkg="vehicle_simulator" type="noisy_state_publisher_with_drift.py" name="noisy_state_vehicle0" output="screen">
            <param name="robot_namespace" value="vehicle0"/>
            <param name="position_drift_per_meter" value="0.3"/>
        </node>
    </group>

    <group ns="vehicle1">
        <node pkg="vehicle_simulator" type="noisy_state_publisher_with_drift.py" name="noisy_state_vehicle1" output="screen">
            <param name="robot_namespace" value="vehicle1"/>
            <param name="position_drift_per_meter" value="0.3"/>
        </node>
    </group>

    <group ns="vehicle2">
        <node pkg="vehicle_simulator" type="noisy_state_publisher_with_drift.py" name="noisy_state_vehicle2" output="screen">
            <param name="robot_namespace" value="vehicle2"/>
            <param name="position_drift_per_meter" value="0.3"/>
        </node>
    </group>
</launch>
